src/controllers/oidc/postCreate.action.ts

```diff
@@ -4,15 +4,19 @@ import { ERROR_SENDING_MAIL_FAILED } from '../../util/messages';
 import AccountService from '../../services/AccountService';
 import { GTM } from '../../util/secrets';
 import { HttpError } from '../../models/Error';
+import { checkPasswordStrength } from '../../util/passwordcheck';
 
 export default async function postCreateController(req: Request, res: Response, next: NextFunction) {
     const { result, error } = await AccountService.isEmailDuplicate(req.body.email);
     const alert = { variant: 'danger', message: '' };
+    const passwordStrength = checkPasswordStrength(req.body.password);
 
     if (result) {
         alert.message = 'An account with this e-mail address already exists.';
     } else if (error) {
         alert.message = 'Could not check your e-mail address for duplicates.';
+    } else if (passwordStrength != 'strong') {
+        alert.message = 'Please enter a strong password.';
```

Comment:
@JustinaMary Please also provide a help text with the password rules in the template. 
```
<p class="small text-muted">Lorem ipsum</p>
```
This should be proper styling under the inputs I guess. Also mind that there is probably input validation on the password field.

Reply:
Sure

Reply:
Done